src/ustreamer/device.c

```diff
@@ -85,6 +86,27 @@ static const char *_io_method_to_string_supported(enum v4l2_memory io_method);
 #define _D_XIOCTL(...)	us_xioctl(_RUN(fd), __VA_ARGS__)
 
 
+void us_v4l2_buffer_init(enum v4l2_buf_type type, struct v4l2_buffer *buf) {
```

Comment:
sorry for my mislead here, maybe we should mark those 3 function as `static` and add a underscore to the function name, like: `static void _us_v4l2_buffer_init(enum v4l2_buf_type type, struct v4l2_buffer *buf) {`

since we won't use those function in another .c here, i think we should make them local to this file

Reply:
It should be like `static void _v4l2_buffer_init(device_t *dev, struct v4l2_buffer *buf);`. `us` is used only for "public" functions.

Reply:
i fixed it